fbfmaproom/fbfmaproom.py

```diff
@@ -200,6 +200,17 @@ def format_enso(x):
         return "El Niño"
     assert False, f"Unknown enso state {x}"
 
+def format_nma_bady(x):
+    if np.isnan(x):
+        return ""
+    if np.isclose(x, 3):
+        return "Moderate Impact"
+    if np.isclose(x, 2):
+        return "Normal"
+    if np.isclose(x, 1):
+        return "High Impact"
```

Comment:
This order seems wrong. Normal shouldn't be in between moderate and high, it should be lower than moderate. Is this the order Dante said to use?